docs/about/introduction.mdx

```diff
@@ -21,11 +22,11 @@ Meshtastic® is a project that enables you to use inexpensive LoRa radios as a l
 
 ## How it works
 
-The underlying technology, LoRa, is a long range radio protocol available to most regions without requiring certification like HAM operators.
+The underlying technology, LoRa, is a long range radio protocol available to most regions without requiring additional licensing or certification, like HAM operators.
 
-The radios automatically rebroadcast messages they receive in order to create a mesh network so that everyone in the group can receive messages - even from the furthest member. Depending on settings used, the Meshtastic mesh can sustain up to 80 device nodes.
+The radios automatically link up and rebroadcast messages they receive in order to create a mesh network so that everyone in the group can receive messages - even from the furthest member. Depending on settings used, the Meshtastic mesh can sustain up to 80 device nodes.
```

Comment:
Don't like the term "link upp" I think we can leave this part as-is

Reply:
accepted